nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -42,11 +42,42 @@ collision_monitor:
     FootprintApproach:
       type: "polygon"
       action_type: "approach"
-      footprint_topic: "/local_costmap/published_footprint"
+      # footprint_topic: "/local_costmap/published_footprint"
```

Comment:
Commented out?

Reply:
This params file is currently modified to run the CM example node with velocity_polygon (for ease of testing without any changes). To discuss if we want to use velocity_polygon by default for the example node, or else I will revert this comment and remove "FootprintApproach" from the `polygons` too.

Reply:
Dont remove anything that was already here.

Reply:
reverted and added a sample for `velocity_polygon` type(not used by default)